src/plugins/intel_gpu/tests/test_cases/gather_gpu_test.cpp

```diff
@@ -6,53 +6,249 @@
 
 #include <intel_gpu/primitives/input_layout.hpp>
 #include <intel_gpu/primitives/gather.hpp>
+#include "ngraph/runtime/reference/gather.hpp"
 
 #include <cstddef>
 
 using namespace cldnn;
 using namespace ::tests;
 
+#define GATHER8_TEST_4D_FORMATS \
+    format::type::bfyx, format::type::b_fs_yx_fsv4, format::type::b_fs_yx_fsv16, format::type::b_fs_yx_fsv32
+#define GATHER8_TEST_5D_FORMATS format::type::bfzyx, format::type::b_fs_zyx_fsv16, format::type::b_fs_zyx_fsv32
+
```

Comment:
Could you add double blocked formats too?

Reply:
Enabled all format.